File: src/main/java/com/manifestors/shinrai/client/event/events/rendering/Rendering2DEvent.java
package com.manifestors.shinrai.client.event.events.rendering;

import com.manifestors.shinrai.client.event.ConstantEvent;
import lombok.AllArgsConstructor;
import lombok.Getter;
import net.minecraft.client.gui.DrawContext;

@Getter
@AllArgsConstructor
public class Rendering2DEvent extends ConstantEvent {

    private final DrawContext context;
    private int width;
    private int height;

}

File: src/main/kotlin/com/manifestors/shinrai/client/event/events/rendering/Rendering2DEvent.kt
/*
 *
 *  * Copyright © 2026 Manifestors Open Source
 *  * License: GPL-3.0
 *  *
 *  * All code in this project is the property of the Manifestors Open Source team
 *  * and its contributors. If you use this code in any project, please provide proper attribution
 *  * and release your project under the GPL-3.0 license as well.
 *  *
 *  * For more details, see: https://www.gnu.org/licenses/gpl-3.0.en.html
 *
 */

package com.manifestors.shinrai.client.event.events.rendering

import com.manifestors.shinrai.client.event.ConstantEvent
import net.minecraft.client.gui.DrawContext

class Rendering2DEvent(
    val context: DrawContext,
    val width: Int,
    val height: Int
) : ConstantEvent()
